ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/UIConstant.kt

```diff
@@ -20,6 +20,8 @@ internal object UIConstant {
     // But lower than 1.0f
     const val halfWeight = 0.5f
 
+    val closeButtonSize = 24.dp
```

Comment:
FFTI but should we use this to set the size of the close button as well? I see this is set in the drawable, but maybe it's worth setting it on code to make sure it matches?

Reply:
good idea!